### PR TITLE
Automatic full restore for Serverless clusters.

### PR DESCRIPTION
**Pull request by ianjevans**
Fixes DOC-3711.

### CODE REVIEW

File: cockroachcloud/backups-page.md [hunk 1]
```diff
@@ -236,3 +258,17 @@ Or [change the existing table's name](../{{site.versions["stable"]}}/rename-tabl
...
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+## Restore a cluster
+
+Find the cluster backup you want to restore, and click **Restore**.
+
+Performing a restore will cause your cluster to be unavailable for the duration of the restore. All current data is deleted, and the cluster will be restored to the state it was in at the time of the backup. There is no automatic incremental backups, and no automatic database or table level backups.
+
+You can [manage your own backups](run-bulk-operations.html), including incremental, database, and table level backups. To perform manual backups, you must configure either a [`userfile`](../{{site.versions["stable"]}}/use-userfile-for-bulk-operations.html) location or a [cloud storage location](../{{site.versions["stable"]}}/use-cloud-storage-for-bulk-operations.html), which requires [billing information](billing-management.html) for your organization.
```
Comment (livlobo) [inline cockroachcloud/backups-page.md:272]: Minor nit: Can we add this phrase here as well:  "Once you enter billing information, even if you don't set a spend limit, ... "

**Review by livlobo — APPROVED** [review]
LGTM

File: cockroachcloud/backups-page.md [hunk 2]
```diff
@@ -1,28 +1,42 @@
...
+</div>
 
-Cockroach Labs runs [full backups](../{{site.versions["stable"]}}/take-full-and-incremental-backups.html#full-backups) daily and [incremental backups](../{{site.versions["stable"]}}/take-full-and-incremental-backups.html#incremental-backups) hourly for every {{ site.data.products.db }} cluster. The full backups are retained for 30 days, while incremental backups are retained for 7 days.
+
+This page describes the **Backups** page and how to restore your data.
 
 The backups that Cockroach Labs runs for you can be viewed on the [Backups page](#backups-page).
 
+<section class="filter-content" markdown="1" data-scope="serverless">
+Cockroach Labs runs [full backups](../{{site.versions["stable"]}}/take-full-and-incremental-backups.html#full-backups) hourly for every {{ site.data.products.serverless }} cluster. The full backups are retained for 30 days.
```
Comment (kathancox) [inline cockroachcloud/backups-page.md:19]: For Serverless, should this line be "Cockroach Labs runs [full cluster backups] hourly...." cc @livlobo

**Review by kathancox — APPROVED** [review]
One small nit/question, otherwise LGTM! Thanks for updating this Ian!

File: cockroachcloud/backups-page.md (hunk 1 above)
Comment (ianjevans) [reply cockroachcloud/backups-page.md:272]: Done.

**Review by gemma-shay — CHANGES_REQUESTED** [review]

**Review by gemma-shay — APPROVED** [review]

**Pull request merged**